### PR TITLE
Remove XDG in favor for Zoom's ~/Documents hardcode

### PR DESCRIPTION
**Pull request by TheEvilSkeleton**
I'm going to send my PRs in this branch from now on, so pushing and testing *should* be easier

Fix https://github.com/flathub/us.zoom.Zoom/issues/208

### CODE REVIEW

**Comment by TheEvilSkeleton** [comment]
Fix https://github.com/flathub/us.zoom.Zoom/issues/208

**Comment by TheEvilSkeleton** [comment]
@Exalm when you have time, please test it out because I am using the English locale so it wouldn't affect me.

File: us.zoom.Zoom.json [hunk 1]
```diff
@@ -11,10 +11,14 @@
         "--socket=pulseaudio",
         "--share=network",
         "--device=all",
-        "--filesystem=home"
+	"--filesystem=~/Documents/Zoom:create",
```
Comment (Exalm) [inline us.zoom.Zoom.json:14]:
```suggestion
        "--filesystem=~/Documents/Zoom:create",
```

But really I already know it works, we have exactly same override there.

Comment (TheEvilSkeleton) [reply us.zoom.Zoom.json:14]: No really sure where you found that. Seeing in my repo, there's no problem: https://github.com/TheEvilSkeleton/us.zoom.Zoom/blob/beta/us.zoom.Zoom.json#L14

Comment (TheEvilSkeleton) [reply us.zoom.Zoom.json]: Weird. I wasn't able to see this issue on my end, Thank you

**Comment by Exalm** [comment]
Checked, of course it works. I mean, this is the same thing I did, I don't know what you expected results to be 🤷️

**Comment by TheEvilSkeleton** [comment]
It's always good to double check.

I think it's good to go now

**Comment by barthalion** [comment]
Something wrong happened here with git history.

**Comment by TheEvilSkeleton** [comment]
Are there any merge conflicts? I don't know where to check

**Comment by TheEvilSkeleton** [comment]
I'll just submit a new MR.

**Pull request closed without merging**